### PR TITLE
Reorder tools for RNA Structure Prediction.

### PR DESCRIPTION
**Pull request by davebx**

### CODE REVIEW

File: files/galaxy/usegalaxy.org/var/shed_tool_conf.xml [hunk 1]
```diff
@@ -3550,36 +3550,36 @@
...
-        <id>toolshed.g2.bx.psu.edu/repos/tyty/structurefold/iterative_map_pipeline/1.0</id>
+        <id>toolshed.g2.bx.psu.edu/repos/tyty/structurefold/react_cal_pipeline/1.0</id>
         <version>1.0</version>
     </tool>
-    <tool file="toolshed.g2.bx.psu.edu/repos/tyty/structurefold/7bb98e9296e9/structurefold/reactivity_cal/reactivity_calculation.xml" guid="toolshed.g2.bx.psu.edu/repos/tyty/structurefold/react_cal_pipeline/1.0">
+    <tool file="toolshed.g2.bx.psu.edu/repos/tyty/structurefold/7bb98e9296e9/structurefold/predict/predict_RNAs.xml" guid="toolshed.g2.bx.psu.edu/repos/tyty/structurefold/predict_pipeline/1.0">
       <tool_shed>toolshed.g2.bx.psu.edu</tool_shed>
         <repository_name>structurefold</repository_name>
         <repository_owner>tyty</repository_owner>
         <installed_changeset_revision>7bb98e9296e9</installed_changeset_revision>
```
Comment (martenson) [inline files/galaxy/usegalaxy.org/var/shed_tool_conf.xml]: how can changeset stay the same when the tool changes?

**Comment by martenson** [comment]
:+1:

**Comment by natefoo** [comment]
Is there a reason not to do this in `integrated_tool_panel.xml`, or is this just for the case of having things show up correctly if `itp` is destroyed?

**Comment by davebx** [comment]
It is for exactly that scenario.

**Comment by natefoo** [comment]
K.

**Pull request merged**